Src/FluentAssertions/Common/MethodInfoExtensions.cs

```diff
@@ -24,10 +24,7 @@ internal static bool IsAsync(this MethodInfo methodInfo)
         internal static IEnumerable<TAttribute> GetMatchingAttributes<TAttribute>(this MemberInfo memberInfo, Expression<Func<TAttribute, bool>> isMatchingAttributePredicate)
             where TAttribute : Attribute
         {
-            var customAttributes = memberInfo.GetCustomAttributes(
-                typeof(TAttribute), false)
-                .Cast<TAttribute>()
-                .ToList();
+            var customAttributes = memberInfo.GetCustomAttributes<TAttribute>(false).ToList();
```

Comment:
I would recommend to always add parameter name for boolean parameters.
For reviewer (like me just now) it is unclear what `false` does mean.

Reply:
I agree. 
I'll create another PR with changes through out the code base.